RELEASING.md

```diff
@@ -12,7 +12,11 @@ To bump all (execpt the `/build` module's) dependencies run:
 
 1. Verify if [wizard](https://app.signalfx.com/#/integrations/go-tracing/description)
    and [official docs](https://docs.splunk.com/Observability/gdi/get-data-in/application/go/get-started.html)
-   needs any adjustments. Contact @signalfx/gdi-docs team if needed.
+   needs any adjustments.
+   Create a Pull Request with documentation updates in
+   [splunk/public-o11y-docs](https://github.com/splunk/public-o11y-docs/tree/main/gdi/get-data-in/application/go)
+   if necessary.
+   Contact @signalfx/gdi-docs team if needed.
```

Comment:
That'd be @splunk/gdi-docs now.

Reply:
https://github.com/orgs/splunk/teams/gdi-docs

Reply:
```suggestion
   Contact @splunk/gdi-docs team if needed.
```